### PR TITLE
Degville add-model swap

### LINKED ISSUE #1019: 2.0 'create-model'=>'add-model'
The juju create-model command for 2.0 is now 'add-model' instead. All existing references and examples need to be updated, plus any contextual text which talks about 'creating' models.

### PR DESCRIPTION
**Pull request by degville**
fixes #1019

### CODE REVIEW

**Comment by evilnick** [comment]
conflicts - we don't like conflicts...

did you remember to git fetch upstream master?

**Comment by pmatulis** [comment]
also, don't touch commands.md and release-notes stuff

**Comment by degville** [comment]
I had edited both command.md (renamed argument and moved it
alphabetically), and the temp-release-notes.md as it referenced
create-models.

**Comment by evilnick** [comment]
@degville the commands page doesn't matter. Generally, we don't edit it because the edits get lost. the text there is automatically generated* from Juju help, which makes sure it is up to date.

The release notes are the release notes, and its probably not worth changing those. However, as another release is imminent, they are likely to be overwritten shortly too.

(*automatically when I run the special script, which I will have to update now we changed the format of the help text)

**Comment by evilnick** [comment]
@juju/docs Let this be a lesson to all - fetch before you branch!

**Comment by evilnick** [comment]
\o/

File: src/en/models.md [hunk 1]
```diff
@@ -4,6 +4,22 @@ TODO: Review again soon (created: April 2016)
 
 # Models
 
+<<<<<<< HEAD
```
Comment (evilnick) [inline src/en/models.md]: you missed a bit. basically, delete from here to ======= below

File: src/en/models.md [hunk 2]
```diff
@@ -16,11 +32,12 @@ and management purposes.
 The '**default**' model however is ready for immediate use, and multiple 
 subsequent models can also be created. 
 See [Creating a model](./models-creating.html).
+>>>>>>> upstream/master
```
Comment (evilnick) [inline src/en/models.md]: and this line

**Comment by evilnick** [comment]
:cake: :beers:

**Pull request merged**